### ISSUE TITLE
cannot integrate uxcam on ionic project

### ISSUE THREAD

**Issue by LaloCastilla** [uxcam/cordova-uxcam#12]
I have followed the instructions but after executing the following lines it shows me on an alert: 'Class not found error'.

`UXCam.startWithKey('d9f707635967f38', function () { alert('started') }, function (err) { alert(err) });`

Im trying on ionic cordova 6.4.0 platform cause trying in 7.0.0 doesnt recognize even the uxcam plugin file.

Ionic:

   ionic (Ionic CLI)  : 4.1.1 (C:\Users\Eduardo\AppData\Roaming\npm\node_modules\ionic)
   Ionic Framework    : ionic-angular 3.8.0
   @ionic/app-scripts : 3.2.0

Cordova:

   cordova (Cordova CLI) : 8.0.0
   Cordova Platforms     : android 6.4.0, ios 4.5.4
   Cordova Plugins       : cordova-plugin-ionic-keyboard 2.1.2, cordova-plugin-ionic-webview 1.2.1, (and 11 other plugins)

System:

   Android SDK Tools : 26.1.1 (C:\Users\Eduardo\AppData\Local\Android\Sdk)
   NodeJS            : v8.11.3 (C:\Program Files\nodejs\node.exe)
   npm               : 5.10.0
   OS                : Windows 10

**Comment by LaloCastilla** [comment]
anyone?

**Comment by theanilpaudel** [comment]
How are you importing the UXCam module?
First, make sure your `package.json` contains
`"cordova-uxcam": "^2.3.7"`
and then import it like this
`declare var UXCam:any;`
and use it like this
`UXCam.startWithKey('d9f707635967f38',function(){
        alert('started')
      },function(error){
        alert('error')
      });`

**Comment by wcustodio** [comment]
Same problem here. Any solutions?

**Comment by susanuxcam** [comment]
Hey @wcustodio do you still have this problem?

**Comment by MarcAndreC** [comment]
I have same problem in android studio when trying to compile. : capacitor-cordova-android-plugins\src\main\java\com\uxcam\cordova\UXCam.java:23: error: cannot find symbol
            com.uxcam.UXCam.startNewSession();

**Comment by susanuxcam** [comment]
Hey @Stylemayster, we've already reported it to our developers. We will let you know as soon as it's fixed!

**Comment by kveber** [comment]
anyone get some feedback about this error or found some alternative?

**Comment by alexsafe** [comment]
definitely still an issue

**Comment by susanuxcam** [comment]
Hey everyone, it's fixed in the latest cordova sdk 3.4.2 - can you try it out?

**Comment by sbellver** [comment]
Same error here

<img width="913" alt="image" src="https://user-images.githubusercontent.com/8066456/195996253-8387ba15-9ddb-4afc-867a-8647cfdfad98.png">

I'm on ionic + vue

After `npm install cordova-uxcam`and `ionic cap update && ionic cap copy && ionic cap sync` :

<img width="864" alt="image" src="https://user-images.githubusercontent.com/8066456/195996328-8fbd4449-2c95-47a9-8a48-04c4dc41ce90.png">

--- EDIT ---

Seems the last error is related with @capacitor-community/uxcam , no Cordova-uxcam ; I remove it and cap works, but general error persists:

```
 ERROR  Failed to compile with 1 error                                                                                                6:06:01 PM

 error  in src/main.ts:101:19

TS2307: Cannot find module 'cordova-uxcam' or its corresponding type declarations.
     99 |
    100 | /* UX Cam */
  > 101 | import UXCam from 'cordova-uxcam';
        |                   ^^^^^^^^^^^^^^^
    102 | declare let UXCam:any;
    103 |
    104 | UXCam.optIntoSchematicRecordings();

 ERROR  Error: Build failed with errors.
```

--- EDIT ---

I can use declare var instead let or const with:

```
/* UX Cam */
// eslint-disable-next-line
declare var UXCam:any;
```

But same result

<img width="1027" alt="image" src="https://user-images.githubusercontent.com/8066456/195997213-cbf081b3-73ae-4438-808b-ca64e4beea0e.png">

-- UPDATE ---

Well, I add 

```
/* UX Cam */
// eslint-disable-next-line
declare var UXCam:any;
```
In main.ts

Then go to App.vue and add the same, plus 
```
UXCam.optIntoSchematicRecordings() //To enable session video recording on iOS 

    const configuration = {
      userAppKey: 'YOUR API KEY',
      enableMultiSessionRecord: true,
      enableCrashHandling: true,
      enableAutomaticScreenNameTagging: false,
      enableImprovedScreenCapture: true,
    }

    UXCam.startWithConfiguration(configuration);
  },
```

at my created() section; obviously with my API KEY

No error, but no sessions.
